### DISCUSSION TITLE
Introducing our new Slack community

### DISCUSSION THREAD

**Discussion by owenrumney** [aquasecurity/tfsec#1150] [Announcements]
We're happy to share that we now have a place to chat and discuss tfsec related topics - on Slack! To join - visit slack.aquasec.com to get an invitation.

You probably wonder what's between this new Slack and GitHub Discussions. GitHub Discussions is still available for Announcements, Q&A and Help, and project development related discussions. Slack is being added as a more informal and flexible way to communicate, based on your feedback. If you need help, GitHub Discussions is still the preferred place to ask, and also for sharing knowledge with other users, but if you feel more comfortable reaching out in chat - please do! The same team of maintainers from GitHub will be monitoring Slack as well.

The Slack workspace is under aquasecurity.slack.com (but remember to get an invite first) and includes channels for every key open source project we maintain, including of course tfsec. Feel free to explore and join other channels and conversations, and also feel free to reach out to us maintainers personally - we're here to talk :)

Thanks for being part of our community, and hope to see you on Slack soon!

**Comment by liamg** [comment]
Invites available [here](https://slack.aquasec.com/)
